File: src/B_5554/B_5554.java
package B_5554;

import java.io.*;
import java.util.*;

public class B_5554 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int t = 0;
        for (int i = 0; i < 4; i++) {
            t += Integer.parseInt(br.readLine());
        }
        System.out.println(t/60);
        System.out.println(t%60);
    }
}

File: src/B_5554/B_5554.kt
package B_5554

import java.io.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){
    var t = 0
    for (i in 0..3) {
        t += readLine().toInt()
    }
    println(t / 60)
    println(t % 60)
}